src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -73,6 +73,7 @@ public without sharing class UTIL_OrgTelemetry_SVC {
         IsEnabled_BDICMT,
         IsEnabled_CustomizableRollups,
         IsEnabled_DefaultGAU,
+        IsEnabled_PaymentAllocation,
```

Comment:
Should this be `IsEnabled_PaymentAllocations` (plural) or is the singular better? The parameter description in the XML has it as plural, so I had the feeling the actual parameter name should be plural as well.